sigs.yaml

```diff
@@ -1026,9 +1026,6 @@ sigs:
       name: Vince Prignano
       company: Red Hat
     tech_leads:
-    - github: CecileRobertMichon
```

Comment:
@CecileRobertMichon given the outstanding work you did in our community, I think that you should add yourself to emeritus_leads

Reply:
I missed this was a thing! Done